app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -67,3 +67,13 @@ fun Date.toRealmInstant(): RealmInstant {
 }
 
 fun Date.isToday(): Boolean = Date().let { now -> year() == now.year() && month() == now.month() && day() == now.day() }
+
+fun View.toggleChevron(
+    isCollapsed: Boolean,
+    collapsedAngle: Float = -180.0f,
```

Comment:
This magic value `-180` is already used in [item_setting_account.xml](https://github.com/Infomaniak/android-mail/pull/132/files#diff-f3a9cd0bcd93546a3756f7819c6a9a6020c1c9b8d803da7f55a938b3606d7c0f)